envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -69,6 +70,11 @@ class EnvoyContainer(
         )
     }
 
+    fun addDnsEntry(host: String, ip: String) {
```

Comment:
What about cleanup? Maybe we should clean up those entries per test class/method. WDYT?

Reply:
added